README.md

```diff
@@ -35,6 +35,21 @@ Edgehog Device Runtime is a reference implementation of
 Astarte interfaces describe how data are exchanged with the remote instance, and what kind of
 features are implemented.
 
+## Configuration file
+
+Edgehog Device Runtime can be configured using a [TOML](https://en.wikipedia.org/wiki/TOML) located either in $PWD/config.toml or /etc/edgehog/config.toml, or in a custom path, run `cargo run -- --help` for more informations.
+
+Example configuration:
+```toml
+credentials_secret = "YOUR_CREDENTIAL_SECRET"
+device_id = "YOUR_UNIQUE_DEVIDE_ID"
+pairing_url = "https://api.astarte.example.com/pairing"
+realm = "examplerealm"
+interfaces_directory = "/usr/share/edgehog/astarte-interfaces/"
+state_file = "/var/lib/edgehog/state.json"
+download_directory = "/var/tmp/edgehog-updates"
```

Comment:
let's keep ourself consistent, do we want to write path with our without trailing `/`?